src/qml/AboutPage.qml

```diff
@@ -29,6 +30,19 @@ Flickable {
         id: diskUsage
     }
 
+    Process {
+        id: process
+        onReadyRead: uptime.text = readAll();
+    }
+
+    Timer {
+        interval: 1000
+        repeat: true
+        triggeredOnStart: true
+        running: true
+        onTriggered: process.start("/bin/cat", [ "/proc/uptime" ]);
```

Comment:
I'm not *opposed* to the idea of having a low level "start command" interface in the style of this "Process" object but if it's only to call `cat` this is completely overkill and we should rather have an API to read a file. 

I'd rather extend https://github.com/AsteroidOS/qml-asteroid/blob/master/src/utils/src/fileinfo.h with something like (untested):

```
    Q_INVOKABLE QString contentOf(const QString fileName);
```

```
QString FileInfo::contentOf(const QString fileName)
{
    QFile file(fileName);
    if(!file.open(QIODevice::ReadOnly)) {
        return "";
    }

    QTextStream in(&file);
    QString content = in.readLine();
    file.close()
    return content;
}
```

Then we don't have to spin up new processes and we don't need to refer to bash script concepts like cat.

Reply:
@magenfire also noted this and has a C function that does the uptime reading.
In the meanwhile, i got a working version of reading the uptime using Qprocess. But i found no replacemnt use for calling a cli command in settings / about page context.
Converting to draft for a while to either come up with something or discard the QProcess involvement.

Reply:
Yeah, I agree that ideally we would even call the `sysinfo` syscall, convert the result into a sensible type and give the timestamp to the QML code without having to manipulate any string in between.